samples/orbit-2-stocklist/src/main/java/com/babylon/orbit2/sample/stocklist/streaming/stock/Stock.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Babylon Partners Limited
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.babylon.orbit2.sample.stocklist.streaming.stock
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class Stock(
+    val itemName: String,
+    val name: String,
+    val bid: String,
+    val ask: String,
+    val timestamp: String
+) : Parcelable
```

Comment:
I realise it's just a sample but what are your thoughts on using `@SerializedName` here